images/linux/scripts/helpers/install.sh

```diff
@@ -50,6 +54,19 @@ verlte() {
     [  "$1" = "$sortedVersion" ]
 }
 
+verify_http_status_code()
+{
+    URL="$1"
+    http_code=$(curl -sL -o out.html -w '%{http_code}' $URL)
```

Comment:
why do you need out.html? This should work fine
```suggestion
    http_code=$(curl -sL -o /dev/null -s -w "%{http_code}\n" $URL)
```